examples/classifier/CL1_compute_dtw_distance.py

```diff
@@ -0,0 +1,33 @@
+# Standard library imports
+import sys
+new_path = sys.path[0][:-len("/examples/timeseries")]
+sys.path.append(new_path)
+import warnings
+warnings.filterwarnings(("ignore"))
+
+# Third party imports
+# /
+
+# Local application imports
+from scifin import timeseries as ts
+from scifin import classifier as cl
+
+# Build two time series of AR(1) type.
+rs1 = ts.auto_regressive(start_date="2000-01-01", end_date="2020-01-01", frequency='M', start_values=[1.],
+                         cst=1., order=1, coeffs=[0.3], sigma=0.1)
+
+rs2 = ts.auto_regressive(start_date="2000-01-01", end_date="2020-01-01", frequency='M', start_values=[1.],
+                         cst=1., order=1, coeffs=[0.35], sigma=0.1)
+
+# Plot
+ts.multi_plot([rs1, rs2])
+
+# Display distance information
+print()
+print("COMPUTING DISTANCES:")
+print("====================")
+print("Euclidean distance: \t\t\t\t\t", cl.euclidean_distance(rs1, rs2))
+print("Dynamic Time Warping distance with mode 'abs': \t\t", cl.dtw_distance(rs1, rs2, mode='abs'))
```

Comment:
I would usually prefer using f-string in print statement, e.g.
`print(f"Euclidean distance:      {cl.euclidean_distance(rs1, rs2)}")`

Reply:
Thanks for the tip. It may be natural to use this formatting indeed.